index.html

```diff
@@ -1171,7 +1171,36 @@ <h3>Malicious Issuers and Verifiers</h3>
     </section>
 
     <section class="informative">
-      <h3>Multistatus Correlation</h3>
+      <h3>Status List Monitoring</h3>
+
+      <p>
+Once a <a>verifier</a> knows of a status list and entry index that is
+associated with a specific <a>holder</a>, it becomes possible for that
+<a>verifier</a> to see updates to that status entry as long as the
+status list continues to be updated. This is useful to a <a>verifier</a> that
+needs to understand when a particular <a>verifiable credential</a> has changed
+status without asking the <a>issuer</a> directly for status information on
+the specific <a>holder</a> or when interacting with the <a>holder</a> to get
+the latest status information is not possible. The feature can also cause a
+privacy violation if the <a>holder</a> is unaware of the ability for the
```

Comment:
I do not believe this issue has been resolved if the suggested changes have been ignored, because we currently have no documented mechanism for the VC to contain personal information about the holder, but only about the subject.